SMUser/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "urn:ngsi-ld:SMUser:123",
+  "type": "SMUser",
+  "user_id": {
+    "type": "Property",
+    "value": "21098319"
+  },
+  "platform": {
+    "type": "Property",
+    "value": "Twitter"
+  },
+  "username": {

```

Comment:
it is recommended to use camel upper case 'userName' or 'name' directly.

Reply:
Definitely, you're right, thanks